docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -3,99 +3,106 @@ Google Cloud Platform Guide
+and make them automatically available in your Ansible inventory.
 
-Ansible contains modules for managing Google Compute Engine resources, including creating instances, controlling network access, working with persistent disks, and managing
-load balancers.  Additionally, there is an inventory plugin that can automatically suck down all of your GCE instances into Ansible dynamic inventory, and create groups by tag and other properties.
+You may see a collection of other GCP modules that do not conform to this
+naming convention. These are the original modules primarily developed by the
+Ansible community. You will find some overlapping functionality such as with
+the the "gce" module and the new "gcp_compute_instance" module. Either can be
+used, but you may experience issues trying to use them together.
 
-The GCE modules all require the apache-libcloud module which you can install from pip:
+While the community GCP modules are not going away, Google is investing effort
+into the new "gcp_*" modules. Google is committed to ensuring the Ansible
+community has a great experience with GCP and therefore recommends that begin
+adopting these new modules if possible.
+
+
+Introduction
+---------------
+The Google Cloud Platform (GCP) modules require both the `requests` and the
+`google-auth` libraries to be installed.
 
 .. code-block:: bash
 
-    $ pip install apache-libcloud
+    $ pip install google-auth requests
 
-.. note:: If you're using Ansible on Mac OS X, libcloud also needs to access a CA cert chain. You'll need to download one (you can get one for `here <http://curl.haxx.se/docs/caextract.html>`_.)
 
 Credentials
 -----------
+Authenticating with GCP is easy and can be done in multiple ways. There are two
+common options for authenticating to GCP:
 
-To work with the GCE modules, you'll first need to get some credentials in the
+* Service Accounts (Recommended): Use JSON service accounts with specific permissions.
+* Machine Accounts: Use the permissions associated with the GCP Instance you're using Ansible on.
+
+For the following examples, we'll be using service account credentials.
+
+To work with the GCP modules, you'll first need to get some credentials in the
 JSON format:
 
 1. `Create a Service Account <https://developers.google.com/identity/protocols/OAuth2ServiceAccount#creatinganaccount>`_
 2. `Download JSON credentials <https://support.google.com/cloud/answer/6158849?hl=en&ref_topic=6262490#serviceaccounts>`_
 
-There are three different ways to provide credentials to Ansible so that it can talk with Google Cloud for provisioning and configuration actions:
+There are two different ways to provide GCP credentials to Ansible:
 
-.. note:: If you would like to use JSON credentials you must have libcloud >= 0.17.0
-
-* by providing to the modules directly
-* by populating a ``secrets.py`` file
+* by specifying them directly as module params
 * by setting environment variables
 
 Calling Modules By Passing Credentials
```

Comment:
Again, for consistency's sake, make the wording match - above you say "by specifying them directly as module params" so the title of this section should be "Providing Credentials as Module Parameters" or something similar, so the reader can follow the flow easily.

Reply:
Done.